cmd/anonymizer/main.go

```diff
@@ -69,6 +69,7 @@ func main() {
 			}
 
 			for _, span := range spans {
+				span := span
 				writer.WriteSpan(&span)
```

Comment:
this is weird. I think I know what the linter is complaining about, and a better solution is
```suggestion
			for i := range spans {
				writer.WriteSpan(&spans[i])
```

Reply:
I am not familiar with the intention of the code but 
`for _, span := range spans {
	// something with &span
```
looks like the developer wants to use copies of the original item, not explicitly manipulating the items in the array. 
If the intention is to use the items directly then I believe this change makes sense.

Reply:
writer treats the spans as immutable, so the copying is not necessary, only extra memory allocation